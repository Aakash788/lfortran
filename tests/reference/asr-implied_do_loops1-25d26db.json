{
    "basename": "asr-implied_do_loops1-25d26db",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/implied_do_loops1.f90",
    "infile_hash": "68e55531db4ca34fe61a1670dec525834de529c9d8ade7bac01f4046",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-implied_do_loops1-25d26db.stdout",
    "stdout_hash": "d07244b6d6ca20c0e3d5a21868530e107d019a4bea1d938c6ecd7875",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}