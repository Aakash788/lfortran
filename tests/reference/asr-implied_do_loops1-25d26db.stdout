(TranslationUnit
    (SymbolTable
        1
        {
            implied_do_loop1:
                (Program
                    (SymbolTable
                        2
                        {
                            a:
                                (Variable
                                    2
                                    a
                                    []
                                    Local
                                    (ArrayConstant
                                        40
                                        [1, 2, 3, ...., 8, 9, 10]
                                        (Array
                                            (Integer 4)
                                            [((IntegerConstant 1 (Integer 4))
                                            (IntegerConstant 10 (Integer 4)))]
                                            FixedSizeArray
                                        )
                                        ColMajor
                                    )
                                    (ArrayConstant
                                        40
                                        [1, 2, 3, ...., 8, 9, 10]
                                        (Array
                                            (Integer 4)
                                            [((IntegerConstant 1 (Integer 4))
                                            (IntegerConstant 10 (Integer 4)))]
                                            FixedSizeArray
                                        )
                                        ColMajor
                                    )
                                    Save
                                    (Array
                                        (Integer 4)
                                        [((IntegerConstant 1 (Integer 4))
                                        (IntegerConstant 10 (Integer 4)))]
                                        FixedSizeArray
                                    )
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            j:
                                (Variable
                                    2
                                    j
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                ),
                            s:
                                (Function
                                    (SymbolTable
                                        3
                                        {
                                            
                                        })
                                    s
                                    (FunctionType
                                        []
                                        ()
                                        Source
                                        Implementation
                                        ()
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        .false.
                                        []
                                        .false.
                                    )
                                    []
                                    []
                                    [(Print
                                        [(ArrayConstant
                                            12
                                            [1, 2, 3]
                                            (Array
                                                (Integer 4)
                                                [((IntegerConstant 1 (Integer 4))
                                                (IntegerConstant 3 (Integer 4)))]
                                                FixedSizeArray
                                            )
                                            ColMajor
                                        )]
                                        ()
                                        ()
                                    )]
                                    ()
                                    Public
                                    .false.
                                    .false.
                                    ()
                                )
                        })
                    implied_do_loop1
                    []
                    [(SubroutineCall
                        2 s
                        ()
                        []
                        ()
                    )
                    (Print
                        [(ArrayConstant
                            40
                            [1, 2, 3, ...., 8, 9, 10]
                            (Array
                                (Integer 4)
                                [((IntegerConstant 1 (Integer 4))
                                (IntegerConstant 10 (Integer 4)))]
                                FixedSizeArray
                            )
                            ColMajor
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
